### ISSUE TITLE
Should IPv6 addresses include brackets?

### ISSUE THREAD

**Issue by PragmaticCypher** [ssbc/ssb-ref#32]
The IPv6 address format for a Scuttlebutt reference is:
```
net:fcbc:6c66:bcd4:d3b5:2a2a:60b3:9b86:498f:8008~shs:ppdSxn1pSozJIqtDE4pYgwaQGmswCT9y15VJJcXRntI=
```
A better way may be the Uniform Identifier Resource (URI) standard for IPv6 addresses (RFC 3986 section 3.2.2), which uses brackets to separate the address from the port:
```
net:[fcbc:6c66:bcd4:d3b5:2a2a:60b3:9b86:498f]:8008~shs:ppdSxn1pSozJIqtDE4pYgwaQGmswCT9y15VJJcXRntI=
```

This may have better parsing support across programming languages.  For instance, Golang's 'net' package will only parse the URI formatted address.

**Comment by christianbundy** [comment]
I'd like to see this change. I'm not sure whether it's compatible with the [multiserver-address](https://github.com/dominictarr/multiserver-address/) syntax, but if so I *think* this would just be a change to [multiserver](https://github.com/ssbc/multiserver).

**Comment by arj03** [comment]
@christianbundy but it is already used. The ws plugin in multiserver uses [URL](https://github.com/ssbc/multiserver/blob/master/plugins/ws.js#L132) so tests in ssb-invite are failing for me, because it generates an addr with [::] :)

**Comment by christianbundy** [comment]
Right, it's used in the websocket plugin but I don't think it works in the net plugin, does it?

**Comment by arj03** [comment]
No, net plugin is not "affected". So I'm thinking something like fix multiserver-address and then use URL in net as well? Maybe together with your big [PR](https://github.com/ssbc/multiserver/pull/49) :) What do you think?
